DeepSpace/Controllers/ShipController.cs

```diff
@@ -88,16 +89,34 @@ public async Task<MoveShipResponse> Move([FromBody] MoveShipRequest value)
             return response;
         }
 
-            //// PUT api/values/5
-            //[HttpPut("{id}")]
-            //public void Put(int id, [FromBody]string value)
-            //{
-            //}
+        [HttpPut]
+        [ActionName("Repair")]
+        public string Repair([FromBody] RepairShipRequest value)
+        {
+            ShipManager.Repair(value.CommandCode, value.Health);
+            return "Ship repaired";
+        }
 
-            //// DELETE api/values/5
-            //[HttpDelete("{id}")]
-            //public void Delete(int id)
-            //{
-            //}
+
+        [HttpPut]
+        [ActionName("Repair")]
```

Comment:
Can we make these posts with the signature /api/ship/repair/hull and /api/ship/repair/shields?